PhpAmqpLib/Connection/AbstractConnection.php

```diff
@@ -559,26 +559,25 @@ protected function wait_channel($channel_id, $timeout = 0)
                 }
                 continue;
 
-            } else {
```

Comment:
@videlalvaro you can remove the `else` or the `continue` ...in my PR i wanted to remove the `else` because you remove one level.

See this example - it's exactly the same behaviour
```php
while(true){
  if($i === 2){
    // do this
  } else{
    // do something elese
  }
}

// this do the same
while(true){
  if($i === 2){
    // do this
    continue;
  }

  // do something elese
}
```